pkg/retester/testdata/prowconfig/simple.yaml

```diff
@@ -0,0 +1,98 @@
+tide:
```

Comment:
Can we simplify this file?
Are All the fields needed for the test?

Reply:
This test is harder for me so I tried simplify.